src/stylus/components/_tabs.styl

```diff
@@ -2,160 +2,151 @@
 @import '../theme'
 
 /* Theme */
-tabs($material)
-  .tabs__li
+tabs__container($material)
+  background-color: $material.cards
+
+  .icon--left,
+  .icon--right
+    color: $material.icons.inactive
```

Comment:
Where are these styles used? I guess it was for append/prepend icons in 0.17, but looks like it's not necessary any longer